rest-client/src/main/java/org/sagebionetworks/bridge/rest/RestUtils.java

```diff
@@ -239,20 +241,63 @@ public static String getAcceptLanguage(List<String> langs) {
     public static UploadSession upload(ForConsentedUsersApi usersApi, File file) throws IOException {
         checkNotNull(usersApi, "ForConsentedUsersApi cannot be null");
         checkNotNull(file, "File cannot be null");
+
+        UploadRequest request = makeUploadRequestForFile(file);
+        UploadSession session = usersApi.requestUploadSession(request).execute().body();
+        uploadToS3(file, session.getUrl());
+        usersApi.completeUploadSession(session.getId(), false).execute();
```

Comment:
Do you want to pass in the flag to make the call synchronous, for clients that want the session to be completed?